assoc-aggregate/prepare_segments_1.py

```diff
@@ -1,36 +1,34 @@
+# Author: Ash O'Farrell (aofarrel@ucsc.edu)
```

Comment:
It's not clear to me that it is a good idea to have this as a python script and a heredoc in the WDL. It seems like it should be one or the other.

Reply:
How would you feel about me making a Docker image that uses uwgac/master as its base and puts this python script into it, and calls that python script?

It is really useful to have the python script separate from the WDL for debugging purposes, but I don't like this sort of duplication either. It requires I manually copy things over, and even though I set it up for that to be done easily, that just kinda feels like asking for trouble (then again, I'd have to manually update a hypothetical Docker image too, so maybe that doesn't change things).